great_expectations/data_context/data_context.py

```diff
@@ -368,7 +369,9 @@ def _normalize_store_path(self, resource_store):
                 resource_store["base_directory"] = os.path.join(self.root_directory, resource_store["base_directory"])
         return resource_store
 
-    def get_docs_sites_urls(self, resource_identifier=None):
+    def get_docs_sites_urls(
+        self, resource_identifier=None, site_name: str = None
```

Comment:
Two style nits:

1. This docstring should be converted to Google style to be consistent with the rest of the codebase.
2. annotate site_name as Optional[str]

Reply:
Nice catches! Fixing